ios/ReactNativeCameraKit/CKCamera.m

```diff
@@ -13,6 +13,172 @@
 #import "CKCameraOverlayView.h"
 #import "CKMockPreview.h"
 
+NSString * NSStringFromInterfaceOrientation(UIInterfaceOrientation o){
```

Comment:
Code style here:
(You can use apple's docs as a reference) 
https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/DefiningClasses/DefiningClasses.html

- `*` directly precedes var name `NSString *string`
- `var ==  var`, make sure there is a space
- I would suggest `o` becomes `orientation`. More descriptive and easier to read
- Is this supposed to be a function? This doesn't look like a function signature I've ever seen in obj-c. I tried it locally and I can't even call it. I might not be aware of a language feature, but I would prefer we use a more conventional function signature.

```suggestion
NSString *NSStringFromInterfaceOrientation(UIInterfaceOrientation o){
```

Reply:
Yes, this is a function, pure C function, just like `NSStringFromCGRect`.